src/dom.js

```diff
@@ -16,6 +16,35 @@
 
 
 /**
+ * Whether the element is currently contained in the DOM. Polyfills
+ * `document.contains()` method when necessary.
+ * See https://developer.mozilla.org/en-US/docs/Web/API/Node/contains
+ * @param {!Document} doc
+ * @param {!Element} element
+ * @return {boolean}
+ */
+export function documentContains(doc, element) {
+  if (!doc.contains) {
+    return documentContainsPolyfillInternal_(doc, element);
+  }
+  return doc.contains(element);
```

Comment:
Is there a reason to use `document.contains` over `documentElement.contains`? Seems the latter has been supported [everywhere](https://developer.mozilla.org/en-US/docs/Web/API/Node/contains) and we won't have to worry about different browsers anymore.